src/main/java/org/kiwiproject/migrations/mongo/AbstractMongockCommand.java

```diff
@@ -38,4 +46,50 @@ protected void run(Bootstrap<T> bootstrap, Namespace namespace, T configuration)
     }
 
     protected abstract void run(Namespace namespace, MongockRunner mongock);
+
+    // TODO: More of a question, do we want to allow through config to pass in an explicit driver classname instead of relying on auto discovery?
```

Comment:
I prefer the more explicit approach these days, as opposed to "magic" that can be difficult to figure out when it (inevitably) goes wrong for whatever reason...

Reply:
Well... this wasn't an either/or, it was asking about an additional way to explicitly state the driver and if you don't find one.

Reply:
Ah ok...plus it's a TODO, so we can leave it there indefinitely :rofl:

Reply:
Maybe I will do this later as it will allow us to test each of the drivers in the tests without the comment/uncomment process.